lib/cli/cliTools.js

```diff
@@ -66,7 +66,7 @@ function getInstanceName(instanceObjID) {
 function enumInstances(objects, adapter) {
     const startkey = 'system.adapter.' + (adapter ? adapter + '.' : '');
     return enumObjects(objects, 'instance', startkey)
-        .then(data => data.filter(it => it._id.startsWith('system.adapter.' + adapter + '.')));
+        .then(data => data.filter(it => it._id.startsWith(startkey)));
```

Comment:
I'm not sure if we even need this filter call anymore. Shouldn't `enumObjects` already return all ids that start with `startKey`?

Reply:
True

Reply:
We could actively ask the user, but I am approx. 100 % sure somebody will still manage to have multihost with identical names. So better prevent it like before.. at least for now